pkg/scorecard.go

```diff
@@ -92,20 +92,16 @@ func RunScorecards(ctx context.Context,
 		return ScorecardResult{}, err
 	}
 
-	commits, err := repoClient.ListCommits()
-	if err != nil {
-		//nolint:wrapcheck
-		return ScorecardResult{}, err
-	}
-
-	if len(commits) == 0 {
-		//nolint:wrapcheck
-		return ScorecardResult{}, sce.Create(sce.ErrScorecardInternal, "no commits found")
+	var commitSHA string
+	if commits, err := repoClient.ListCommits(); err != nil && len(commits) > 0 {
+		commitSHA = commits[0].SHA
+	} else {
```

Comment:
let's differentiate between the "no commit" error and the case `ListCommits()` returned an error?

Reply:
Done.